docs/howto/manage-resources.rst

```diff
@@ -79,6 +79,11 @@ will be a new revision.
 
     See more: :ref:`ref_commands_upload-resource`
 
+.. admonition:: Best practice
```

Comment:
Per today's meeting: I think this is worth keeping in charmcraft *and* adding to ops :-)

Reply:
As per the above, I've left this in, and also added it in the PR for ops that include things cur from here.